Package path ./runtime.js is not exported from package

`react-refresh` started using the exports field in package.json
https://github.com/facebook/react/commit/2f26eb85d657a08c21edbac1e00f9626d68f84ae

Since then, the following error is thrown:
```
[0] ERROR in ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js 4:23-58
[0] Module not found: Error: Package path ./runtime.js is not exported from package ./node_modules/react-refresh
(see exports field in ./node_modules/react-refresh/package.json)
```

> `react-refresh` started using the exports field in package.json
> 
> https://github.com/facebook/react/commit/2f26eb85d657a08c21edbac1e00f9626d68f84ae
> 
> 
> 
> Since then, the following error is thrown:
> 
> ```
> 
> [0] ERROR in ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js 4:23-58
> 
> [0] Module not found: Error: Package path ./runtime.js is not exported from package ./node_modules/react-refresh
> 
> (see exports field in ./node_modules/react-refresh/package.json)
> 
> ```
> 
> 

I believe this is not published yet?

--- a/packages/webpack-build-scripts/package.json
+++ b/packages/webpack-build-scripts/package.json
@@ -6,7 +6,7 @@
         "test": "exit 0"
     },
     "dependencies": {
-        "@pmmmwh/react-refresh-webpack-plugin": "^0.5.4",
+        "@pmmmwh/react-refresh-webpack-plugin": "^0.5.7",
         "assert": "^2.0.0",
         "autoprefixer": "^10.4.4",
         "css-loader": "^6.7.1",

